README.md

```diff
@@ -166,12 +166,18 @@ Web Chat strives to use responsive design when possible. As part of this, Web Ch
 
 ### Strings
 
-You can alter or add localized strings in `/src/Strings.ts`:
+You can alter or add localized strings in [/src/Strings.ts](src/Strings.ts):
 
 * Add one or more locales (with associated localized strings) to `localizedStrings`
 * Add logic to map the requested locale to the supported locale in `strings`
 * Please help the community by submitting a [pull request](https://github.com/Microsoft/BotFramework-WebChat/pulls).
 
+You can specify locale to use in your website depends on your usage:
+
+* IFRAME: append `&locale=en-us` in query string
```

Comment:
Is this the Bot Framework embed Iframe?

Reply:
Yes, adding new feature inside the IFRAME host.

Reply:
I'm confused by this. Are you talking about customizing locale inside of the WebChat Bot Framework channel? This repo is not oriented towards that channel.

Reply:
I think we should remove this, because we are refraining from using URL parameters in the Iframe.

Reply:
Again I'm confused by the intention here. Is this about the channel or just our IFRAME sample? If the latter, that's just a sample and it shouldn't be referenced in the README.

Reply:
The feature is in the code already. I will update one of the sample in the README.md to surface the feature.